content/css/concepts/colors/terms/rgb/rgb.md

```diff
@@ -13,7 +13,7 @@ CatalogContent:
   - 'paths/full-stack-engineer-career-path'
 ---
 
-Defines the color value by providing three comma separated values. The first value represents the red value, the second represents the green value, and the third represents the blue value.
+The `rgb()` function in CSS defines a color by specifying its components using three comma-separated values. The first value represents the red component, the second value represents the green component, and the third value represents the blue component. These values can be provided as either numeric values ranging from 0 to 255 or as percentage values ranging from 0% to 100%. By adjusting the values for each component, you can create a wide range of colors, allowing for precise control over the appearance of elements in your web design.
```

Comment:
we avoid first- and second-person pronouns in Docs. So remove use of `you` `we` etc